cryptoprotocol/src/main/java/org/cryptimeleon/incentivesystem/cryptoprotocol/benchmark/BenchmarkConfig.java

```diff
@@ -0,0 +1,68 @@
+package org.cryptimeleon.incentivesystem.cryptoprotocol.benchmark;
+
+import lombok.AllArgsConstructor;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.IncentiveSystem;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.Setup;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.IncentivePublicParameters;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.keys.provider.ProviderPublicKey;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.keys.provider.ProviderSecretKey;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.keys.user.UserPublicKey;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.keys.user.UserSecretKey;
+
+/**
+ * A simple data class for a benchmark configuration with different constructors.
```

Comment:
Maybe add a list of components of the benchmark config to the docstring.

Reply:
I don't think this is interesting for a user